containers/Dockerfile_prepare_db_and_params

```diff
@@ -0,0 +1,13 @@
+FROM debian:11
```

Comment:
There is a `aria2` container available at bioconda, could we use it? https://anaconda.org/bioconda/aria2

Reply:
I use some tar and wget commands apart from aria2. I will give it a try